dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -243,22 +245,24 @@ const buildAbTestTrackingAuxiaVariant = (
 	};
 };
 
+/**
+ * Determines the gate version based on one of 2 things:
+ * 1. url query parameter
+ * 2. treatmentType
+ */
 export const getAuxiaGateVersion = (
-	signInGateVersion?: AuxiaGateVersion,
 	userTreatment?: AuxiaAPIResponseDataUserTreatment,
 ): AuxiaGateVersion => {
-	if (signInGateVersion) {
-		return signInGateVersion;
+	let urlParamVersion: string | null = null;
+	if (typeof window !== 'undefined') {
+		const params = new URLSearchParams(window.location.search);
+		urlParamVersion = params.get('auxia_gate_version');
 	}
 
-	const params = new URLSearchParams(window.location.search);
-	const version = params.get('auxia_gate_version');
-
 	if (
-		String(version).toLowerCase().endsWith('v2') ||
-		String(userTreatment?.treatmentType)
-			.toLowerCase()
-			.endsWith('POPUP')
```

Comment:
I noticed this logic was wrong, as it calls toLowerCase!
Instead, we now check for specific values of `treatmentType`, as it's a union of strings